topics/transcriptomics/tutorials/pinpoint-key-pathways-with-heinz/tutorial.md

```diff
@@ -0,0 +1,404 @@
+    - "Analyzing the pinpointed network."
+contributors:
+    - cicozhang
+    - sanneabeln
+---
+
+# Overview
+{:.no_toc}
+
+The human microbiome plays a key role in health and disease. Thanks to comparative metatranscriptomics,
+the cellular functions that are deregulated by the microbiome in disease can now be computationally
+explored. Unlike gene-centric approaches, pathway-based methods provide a systemic view of such
+functions; however, they typically consider each pathway in isolation and in its entirety.
+They can therefore overlook the key differences that (i) span multiple pathways, (ii) contain
+bidirectionally deregulated components, (iii) are confined to a pathway region. To capture these
+properties, computational methods that reach beyond the scope of predefined pathways are needed.
+
+In this tutorial, we will perform a network analysis using [Heinz](https://github.com/ls-cwi/heinz)
+in Galaxy. The data comes from the study [May et al.](https://academic.oup.com/bioinformatics/article/32/11/1678/2240171),
+and we will reproduce some of the computational steps from this study with simplified data and
+parameters to speed up the analysis for the purposes of this tutorial.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+> ### {% icon comment %} Note
+> Your results may be slightly different from the ones presented in this tutorial due to the tool versions or
+> reference data versions or stochastic processes in the algorithms.
+{: .comment}
+
+In this tutorial, we will create a Heinz workflow step by step, as the picture below shows.
+
+![Heinz workflow](../../images/heinz-workflow.png)
+
+# Obtaining and preparing data  [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1344105.svg)](https://doi.org/10.5281/zenodo.1344105)
+
+
+The study [May et al.](https://academic.oup.com/bioinformatics/article/32/11/1678/2240171) includes the computation
+steps starting from the raw RNAseq datasets. The operations that processed raw data into the interpreted data are
+beyond the scope of this tutorial. To learn that, please refer to the relevant topics in the Galaxy training
+material. In this tutorial, we start with the interpreted data, which are KO
+([KEGG Orthology](https://www.genome.jp/kegg/ko.html)) count data. All the data needed for this tutorial are
+available from Zenodo.
+
+## Understanding our input data
+
+According to the study [May et al.](https://academic.oup.com/bioinformatics/article/32/11/1678/2240171),
+dental caries (DC) dataset in this tutorial came from an experiment that comprised of supragingival plaque
+samples collected from all dental surfaces of 36 individuals who had either a caries-positive (disease) or
+a caries-negative (health) oral profile. Each of the 36 samples was sequenced, pre-processed and transformed
+into KO counts. We will use these count data as the starting point to perform the network analysis.
```

Comment:
maybe you can explain a bit more what KO counts are exactly? like show a couple of line from one of the files and tell us what each column means exactly (I was not familiar with this at least)

Reply:
Fixed it!